filebeat/module/haproxy/_meta/kibana/6/dashboard/Filebeat-haproxy-overview.json

```diff
@@ -156,7 +155,7 @@
               "enabled": true,
               "id": "1",
               "params": {
-                "field": "haproxy.client_ip"
+                "field": "haproxy.client.ip"
```

Comment:
Did we change the name of this field?

Reply:
Yes, in #8637, before the module was released, but without checking the dashboard :(